opentelemetry-sdk/CHANGELOG.md

```diff
@@ -1,7 +1,13 @@
 # Changelog
 
+## Deprecation Notice
```

Comment:
should this instead be a "Migration Notice" or "Functionality moved"  or something similar? As this component is moved to another crate without an immediate plan to remove it from the ecosystem entirely?